week-3/A-array-find/exercise2.js

```diff
@@ -0,0 +1,14 @@
+/*
+  You are given an array of names.
+  Using .find(), we'd like to find the first name which starts with A and is longer than 7 letters.
+*/
+
+// write your code here
+findLongNameThatStartsWithA = names =>names.find(x => x[0] === "A" && x.length > 7);
```

Comment:
Well done! This is very close, but there's 1 small problem.

What if we changed the `names` array to `["Rakesh", "alexandra", "Annam", "Mikey", "Karim", "Ahmed"]`? Would the answer still be correct? How can we fix it?